src/scripts/main.js

```diff
@@ -355,6 +355,25 @@ const people = [
 ];
 
 // eslint-disable-next-line no-console
-console.log(people); // you can remove it
+(() => {
+  const dashboard = document.querySelector('.dashboard');
 
-// write your code here
+  if (!dashboard) return;
```

Comment:
It's good that you're checking if 'dashboard' exists before proceeding, but it would be even better if you could provide some feedback to the user or developer in case it doesn't. This could be a console warning or an error message displayed on the page.